app/src/app/components/sidebar/ZoneLockPicker.tsx

```diff
@@ -6,9 +6,13 @@ import * as RadioGroup from '@radix-ui/react-radio-group';
 import {blackA} from '@radix-ui/colors';
 import {useMapStore} from '../../store/mapStore';
 import {ColorPicker} from './ColorPicker';
+import { LockClosedIcon, LockOpen2Icon } from '@radix-ui/react-icons';
 
 export function ZoneLockPicker() {
   const lockedZones = useMapStore(state => state.mapOptions.lockPaintedAreas);
+  const mapDocument = useMapStore(state => state.mapDocument)
+  const numDistricts = mapDocument?.num_districts || 40
```

Comment:
nice

Reply:
We have 39 but this is just for generating the list of zones to lock